src/modules/timetable/containers/calendar.js

```diff
@@ -1,20 +1,52 @@
 import {render} from 'react-dom';
 import {compose} from 'react-komposer';
 
+import { Server } from '../../core/helpers/';
+
 import component from '../components/calendar';
 
+const calendarService = Server.service('/calendar');
+
 const composer = (props, onData) => {
 	let view = location.hash.replace('#', '');
 	if(!['month','agendaWeek','agendaDay'].includes(view)) {
 		view = 'month';
 	}
 
-	let componentData = {
-		events: [],
-		view
-	};
+	let error;
+	const getEvents = new Promise((resolve, reject) => {
```

Comment:
Wrapping this in a new Promise is unnecessary, you could just write `const getEvents = calendarService.find().catch(err => {return []});`